crates/shim/src/lib.rs

```diff
@@ -129,7 +129,13 @@ pub trait Shim {
     type T: Task + Send + Sync;
 
     /// Create a new instance of Shim.
-    fn new(id: &str, namespace: &str, publisher: RemotePublisher, config: &mut Config) -> Self;
+    fn new(
+        runtime_id: &str,
+        id: &str,
```

Comment:
Could you please update docs for this function to clarify the difference between `runtime_id` and `id`?